packages/atomic/src/components/search/atomic-search-interface/atomic-search-interface.tsx

```diff
@@ -446,6 +472,7 @@ export class AtomicSearchInterface
     return [
       this.engine && (
         <atomic-relevance-inspector
+          open={this.openRelevanceInspector}
```

Comment:
Should we conditionally render the atomic-relevance-inspector component depending on the `enableRelevanceInspector` value?